cpp/ps0001/ps0001_soin_cpp.cpp

```diff
@@ -1,15 +1,19 @@
-#include<iostream>
+#include <iostream>
 #include <conio.h>
 using namespace std;
 int main()
 {
-	int x, y, z, f;
+	// prompting users for 3 numbers to add 1/2
+	unsigned short x, y, z, f;
 	cin >> x;
 	cin >> y;
 	cin >> z;
+
+	// outputting the result of the summation process 3/3
 	f = x + y + z;
 	cout << "_______" << endl;
 	cout << f;
 	_getch();
 
-}
\ No newline at end of file
+}
+// great work 😎 8/10 ⭐
```

Comment:
- [x] program is in the correct path
- [x] no syntax errors
- [x] no bugs occurs while execution
- [x] formatted code to be clear and readable
- [ ] commented some phrases for blocks of code